### ISSUE TITLE
Add before/after audio samples to README

### ISSUE THREAD

**Issue by Keeo** [lawl/NoiseTorch#19]
Does this reduce static noise or ambient noise like nvidia's rtx voise?

**Comment by lawl** [comment]
It should, yes. Do you have a suggestion on how to make the readme more clear?

**Comment by kuzavas** [comment]
I would suggest to add a few samples with/without noise reduction. That would be more clearer than any description.

**Comment by lawl** [comment]
Yep, this is a great suggestion and something I definitely want to do, I just haven't gotten around doing it yet. If it's okay with you, I'll re-purpose this issue to remind me of doing that.

**Comment by Keeo** [comment]
Actually I think the readme is missing informatrion about what technology is used. That would also clarify what type of noise it can reduce and give a highlight to core part.

After quickly checking the repo it looks like NoiseTorch is leveriging [noise-suppression-for-voice](https://github.com/werman/noise-suppression-for-voice) and that is built on top of [https://github.com/xiph/rnnoise](https://github.com/xiph/rnnoise).

The first paragrapf from _noise-supression-for-voice_ has already explenation of what type of noise it can reduce:

> The plugin is meant to suppress a wide range of noise origins (from original paper): computer fans, office, crowd, airplane, car, train, construction.

**Comment by lawl** [comment]
It is in the readme: https://github.com/lawl/NoiseTorch#special-thanks-to

**Comment by Keeo** [comment]
Perfect, so no further points from my side. I'll leave this issue open as you hijacked it :smile:

**Comment by Zardoz89** [comment]
I have a record without the filter active and with the filter active. The difference it's impressive. I try to record audio with my acclimatizer working behind me and typing on my mechanical keyboard. All the noise it's removed. I only noticed some residual noise from the keyboard at same time that I'm speaking. The only problem that I noticed its that sometimes sound like it cut's too early my voice or opens too late. I think that I need to tweak more the voice threshold, to fix it.
